emissionsapi/download.py

```diff
@@ -41,12 +41,16 @@ def download():
     os.makedirs(storage, exist_ok=True)
 
     for country in countries:
-        wkt = bounding_box_to_wkt(*country_bounding_boxes[country][1])
+        if country == "all":
```

Comment:
Would specifying no country filter at all not make more sense than introducing a magic string?

Reply:
We do need a filter for other countries than DE anyway .. don't we?

Reply:
@TylerBird, I don't quite understand your comment. What has this patch to do with specifying Germany specifically via country code?

Reply:
@lkiesow The problem is that the setting no countries leads to the default (`['DE']`). I was unsure about changing the default behavior here, but I agree that a magic string is not a nice solution either. Do you have a better idea?

Reply:
We could either use a different default or allow specifications of empty values in the configuration like this:

```yaml
download:
  # List of two-letter country codes specifying the countries to request data
  # for.
  # Default: []
  country:
```

I would lean to a different default since, for development, you usually use a modified configuration anyway and limiting this to Germany by default was just a split-second decision due to our first iteration goals.

Reply:
I changed the default now. If no country is set, it is not filtered by country.